src/software/ai/motion_constraint/motion_constraint_set_builder_test.cpp

```diff
@@ -9,240 +9,242 @@
 #include "software/world/game_state.h"
 
 // This namespace contains all the test parameters
```

Comment:
there seems to be no documentation on why we're disabling all the tests here

Reply:
good point, I'll look into this